### PR TITLE
Only call template additions if they're not nil.

### PR DESCRIPTION
**Pull request by jyaworski**
Fixes https://github.com/puppet-community/modulesync/issues/64

### CODE REVIEW

**Comment by jyaworski** [comment]
Hold on this for now.

**Comment by jyaworski** [comment]
OK, now we're OK.

File: moduleroot/.rubocop.yml [hunk 1]
```diff
@@ -5,9 +5,16 @@ AllCops:
     - vendor/**/*
     - pkg/**/*
     - spec/fixtures/**/*
+    <%- puts @configs.class -%>
```
Comment (igalic) [inline moduleroot/.rubocop.yml]: this is debugging code, and should should go…

File: moduleroot/.rubocop.yml [hunk 2]
```diff
@@ -5,9 +5,16 @@ AllCops:
...
+    <%- unless @configs.nil? -%>
+    <%- if @configs.has_key?('AllCops') -%>
+    <%- if @configs['AllCops'].has_key?('Exclude') -%>
```
Comment (igalic) [inline moduleroot/.rubocop.yml]: if you "simplify" this to:
```erb
<%- if !@configs.nil? and @configs.has_key?('AllCops') and @configs['AllCops'].has_key?'[Exclude'] -%>
```
but, imo, we should just join those keys, so we'd have:
```erb
<%- if !@configs.nil? and @configs.has_key?('AllCops_Exclude') -%>
```

**Comment by jyaworski** [comment]
Changes will be made momentarily.

**Comment by jyaworski** [comment]
@igalic rebased.

**Pull request merged**